### PR TITLE
Complete the OpenAPI update automation

### PR DESCRIPTION
**Pull request by pakrym-stripe**
Point to the real repositories, add messages, titles, branch names and a trigger.

Example: https://github.com/pakrym-stripe/stripe-cli/pull/1

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -2,25 +2,58 @@ name: Publish
 
 on:
   workflow_dispatch: {}
+  push:
+    tags:
+      - v[0-9]+
 
 jobs:
   publish-stripe-mock:
```
Comment (dcr-stripe) [inline .github/workflows/publish.yml:10]: Rename since this is also publishing to stripe-cli?

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -2,25 +2,58 @@ name: Publish
...
         cp openapi/spec3.json stripe-mock/openapi/spec3.json
         cp openapi/fixtures3.json stripe-mock/openapi/fixtures3.json
+
     - name: Create Pull Request
       uses: peter-evans/create-pull-request@v4
       with:
         path: stripe-mock
+        title: OpenAPI Update
+        body: Automated OpenAPI update for https://github.com/stripe/openapi/commit/${{ github.sha }}
+        branch: update-openapi
```
Comment (dcr-stripe) [inline .github/workflows/publish.yml:43]: Wonder if we should qualify this - what happens if there's multiple changes before we have time to merge a PR?

**Review by dcr-stripe — COMMENTED** [review]
LGTM overall, just a few small questions! 

> Point to the real repositories

Should it? Currently it's still pointing to your forks? Or is this just stale PR description?

File: .github/workflows/publish.yml (hunk 2 above)
Comment (pakrym-stripe) [reply .github/workflows/publish.yml:43]: It force-pushes.

Comment (pakrym-stripe) [reply .github/workflows/publish.yml:43]: https://github.com/pakrym-stripe/stripe-cli/pull/1 has an example

**Comment by pakrym-stripe** [comment]
> Currently it's still pointing to your forks

It will always use bot's forks but PRs are opened against the real repos: https://github.com/stripe/openapi/pull/88/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R18

**Review by dcr-stripe — APPROVED** [review]
LGTM!

**Review by dcr-stripe — APPROVED** [review]
LGTM!

**Pull request merged**